### DISCUSSION TITLE
[Script request] Babybuddy

### DISCUSSION THREAD

**Discussion by bachjessen** [community-scripts/ProxmoxVE#217] [Request script]
### Application Name

babybuddy

### Website

https://github.com/babybuddy/babybuddy

### Description

A script to install and configure the Baby Buddy application on a Proxmox virtual machine, providing an easy way to deploy the baby tracking tool.
The script sets up the required dependencies, creates the database, and starts the Baby Buddy server within the Proxmox environment.
It simplifies the deployment process for users who want to run Baby Buddy on a Proxmox host.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Ready for Test, please visit:
https://github.com/community-scripts/ProxmoxVED/issues/280
